docs/using/api.md

```diff
@@ -131,6 +131,41 @@ You can also submit InSAR jobs for scene pairs using [ESA granule IDs](https://s
 }
 ```
 
+## Submitting Sentinel-1 InSAR burst jobs
+You can submit InSAR jobs using pairs of [ESA granule burst IDs](https://sentiwiki.copernicus.eu/web/s1-products#S1Products-BurstIDMapsS1-Products-Burst-ID-Maps){target=_blank} or mosaic lists of burst ID pairs.
```

Comment:
```suggestion
You can submit InSAR jobs using the `INSAR_ISCE_BURST` job type, which takes a single pair of [ESA granule burst IDs](https://sentiwiki.copernicus.eu/web/s1-products#S1Products-BurstIDMapsS1-Products-Burst-ID-Maps){target=_blank}, or use the `INSAR_ISCE_MULTI_BURST` job type, which accepts lists of up to 15 contiguous along-track burst IDs to mosaic together to produce reference and secondary files.
```

Reply:
There may also be merit to splitting the code into two sections, but I think it's okay this way (especially because the single-burst job type is on its way out).